trashtalk/templates/_login.html

```diff
@@ -1,42 +1,43 @@
-<!DOCTYPE html>
-<!--
-To change this license header, choose License Headers in Project Properties.
-To change this template file, chotoolsools | Templates
-and open the template in the editor.
--->
-{%extends '_base.html'%}
-{%block content%}
-<div class="container">
-    <div class="row">
-        <div class="col-lg-4">
-            <h2>Welcome Back!</h2>
-        </div>
-    </div>
-    <div class="form-group">
-        <form action ="/login" method ="POST">
-            <div class="row">
-                <div class="col-sm-4">
-                    <input type="text"
-                           class="form-control"
-                           name="username"
-                           placeholder="Username"
-                           required pattern="{{username_pattern}}"
-                           title="{{username_title}}">
-                </div>
-            </div>
-                <div class="row">
-                    <div class="col-sm-4">
-                        <input class="form-control"
-                               type="password"
-                               name="password"
-                               placeholder="Password"
-                               required pattern="{{password_pattern}}"
-                               title="{{password_title}}">
-                    </div>
-                </div>
-            <input class="btn btn-sm" type="submit" value="Log in">
-            <a href='/'>Cancel</a>
-        </form>
-    </div>
-</div>
+<!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, chotoolsools | Templates
+and open the template in the editor.
+-->
+{%extends '_base.html'%}
+{%block content%}
+<div class="container">
+    <div class="row">
+        <div class="col-lg-4">
+            <h2>Welcome Back!</h2>
+        </div>
+    </div>
+    <div class="form-group">
+        <!--TODO: change action destintation-->
+        <form action ={% url 'login' %} method ="POST">
```

Comment:
Great work but I didn't foresee issues we may have with styling forms. Lets chat about it and see what we want to do here because I've currently implemented ModelForms which is basically a way to make forms in Django without needing to write any HTML forms. It just uses the models and formulates all the fields based on them. 

That's why in the code you edited had `{{ form }}` in them. Of course I just dropped it in there all ugly but I could have added the styles you've implemented too. You can check out what I'm referring to here: 
https://docs.djangoproject.com/en/1.11/topics/forms/modelforms/#modelform

I think right now your approach may be what we need but lets talk about how we want to approach this.